### PR TITLE
Add E2E test on login and collection creation

### PR DESCRIPTION
**Pull request by HelloYeew**
I just add E2E test. Now you can check it. I write a test on both mobile and desktop view (Since we have on both view and it's act differently). You can continue or use this as reference if you want to write on other part of the website.

Co-author: @some-random-idiot

### CODE REVIEW

File: test_e2e/test_create_collection_mobile.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"""Program for running E2E test via Selenium on collection creation on mobile device or small browser screen"""
+
+import time
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+# Configuration on test
+URL = "http://127.0.0.1:8000/"
+browser = webdriver.Chrome('chromedriver.exe')
```
Comment (pontakornth) [inline test_e2e/test_create_collection_mobile.py:9]: But .exe file is for Windows. It does not work on Mac OS or Linux.

File: test_e2e/test_create_collection_desktop.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+import time
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+# Configuration on test
+URL = "http://127.0.0.1:8000/"
+TEST_USERNAME = 'test'
+TEST_PASSWORD = 'peppytest45'
+browser = webdriver.Chrome('chromedriver.exe')
```
Comment (pontakornth) [inline test_e2e/test_create_collection_desktop.py:11]: This does not work on Mac OS as well.

**Review by pontakornth — CHANGES_REQUESTED** [review]
If webdriver has `.exe` extension, it can't work on Mac OS.

File: test_e2e/test_create_collection_desktop.py (hunk 2 above)
Comment (pontakornth) [reply test_e2e/test_create_collection_desktop.py]: The problem is solved in new E2E test file.

**Review by some-random-idiot — APPROVED** [review]

**Review by SainTurDaY27 — APPROVED** [review]

**Review by pontakornth — APPROVED** [review]
I just fixed some issues so I will approve it.

**Review by Siraphop4Nene — APPROVED** [review]

**Pull request merged**